### ISSUE TITLE
[TalkBack] Wrong Swedish translation

### ISSUE THREAD

**Issue by GoogleCodeExporter** [clc/eyes-free#152]
```
What steps will reproduce the problem?
Wrong word for the Enter key when typing text.
1. Talkback on.
2. Open up any text input and insert a new line.
What is the expected output? What do you see instead?
Talkback will say in Swedish öppna, but it should be Enter. The word öppna is 
more like the English word open.

What version of the product are you using? On what operating system?
Latest Talkback 3.2 Beta 1
Android 4.1.1 Jelly bean


Please provide any additional information below.


```

Original issue reported on code.google.com by `christia...@runbox.com` on 21 Nov 2012 at 5:45

**Comment by GoogleCodeExporter** [comment]
Original comment by `alanv@google.com` on 7 Dec 2012 at 6:35
- Changed title: **[TalkBack] Wrong Swedish translation**
- Added labels: **Translation**

**Comment by GoogleCodeExporter** [comment]
Original comment by `alanv@google.com` on 7 Dec 2012 at 6:35

**Comment by GoogleCodeExporter** [comment]
```
Translation has changed since this bug was filed. Please reopen this bug if the 
issue persists.
```

Original comment by `alanv@google.com` on 30 Apr 2013 at 8:05
- Changed state: **Fixed**
